core/state/statedb.go

```diff
@@ -474,6 +474,20 @@ func (s *StateDB) Suicide(addr common.Address) bool {
 	return true
 }
 
+func (s *StateDB) SendAll(addr common.Address) bool {
+	stateObject := s.getStateObject(addr)
+	if stateObject == nil {
+		return false
+	}
+
+	s.journal.append(sendallChange{
+		account: &addr,
+	})
```

Comment:
I think this is subtly wrong. See how the `suicideChange` above has a notion of `prev`. So when it reverts, it sets the `obj.suicided = ch.prev`. This way, it can apply the change two times, roll back one, and still come out correct. 
Your `sendall` doesn't maintain a `prev`. If applied two, and rolled back one time, it would erroneously revert and set `obj.sendalled = false`. 

Now, please do not immediately fix this! First, add a test which demonstrates the error.

Reply:
I will write a test case for it.  `execution-spec-tests` has changed somewhat upstream and I am getting up to speed.